draft-ietf-quic-qlog-h3-events.md

```diff
@@ -459,93 +459,93 @@ HTTPField = {
 ~~~
 {: #httpfield-def title="HTTPField definition"}
 
-### HTTPCancelPushFrame
+### H3CancelPushFrame
 
 ~~~ cddl
-HTTPCancelPushFrame = {
+H3CancelPushFrame = {
     frame_type: "cancel_push"
     push_id: uint64
 }
 ~~~
-{: #http-cancelpushframe-def title="HTTPCancelPushFrame definition"}
+{: #h3-cancelpushframe-def title="H3CancelPushFrame definition"}
 
-### HTTPSettingsFrame
+### H3SettingsFrame
 
 ~~~ cddl
-HTTPSettingsFrame = {
+H3SettingsFrame = {
     frame_type: "settings"
-    settings: [* HTTPSetting]
+    settings: [* H3Setting]
 }
 
-HTTPSetting = {
+H3Setting = {
     name: text
     value: uint64
 }
 ~~~
-{: #httpsettingsframe-def title="HTTPSettingsFrame definition"}
+{: #h3settingsframe-def title="H3SettingsFrame definition"}
 
-### HTTPPushPromiseFrame
+### H3PushPromiseFrame
 
 ~~~ cddl
-HTTPPushPromiseFrame = {
+H3PushPromiseFrame = {
     frame_type: "push_promise"
     push_id: uint64
     headers: [* HTTPField]
 }
 ~~~
-{: #httppushpromiseframe-def title="HTTPPushPromiseFrame definition"}
+{: #h3pushpromiseframe-def title="H3PushPromiseFrame definition"}
 
-### HTTPGoAwayFrame
+### H3GoAwayFrame
```

Comment:
NIT: case is not consistent GoAway in the title vs Goaway in the Frame definition